python/lsst/pipe/drivers/constructCalibs.py

```diff
@@ -699,6 +713,43 @@ def write(self, butler, exposure, dataId):
         self.log.info("Writing %s on %s" % (dataId, NODE))
         butler.put(exposure, self.calibName, dataId)
 
+    def makeCameraImage(self, butler, dataId, calibs):
```

Comment:
Hmm... after reading a bit more, I realized bulter is being passed around in multiple places in `constructCalibs`...  I thought we are discouraged from doing so inside `Tasks`?